data/mods/Perk_melee/menu.json

```diff
@@ -509,5 +509,11 @@
     "id": "TALK_PERK_MENU_FAIL",
     "dynamic_line": "You don't have a perk point, or meet the prerequisites for this perk.",
     "responses": [ { "text": "Go Back.", "topic": "TALK_PERK_MENU_MAIN" }, { "text": "Quit.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_MA_PERK_MENU_HELP",
+    "dynamic_line": "XP is gained by killing enemies, either by melee or ranged combat, with tougher enemies giving more XP.  You'll gain a perk point every time you level up, which can be refunded at any time through this menu.  Most perks are completely passive, and apply their effects automatically on movement, combat or waiting.\n\nPerks and techniques cannot be refunded or disabled once learned, consider your choices carefully.",
```

Comment:
`You'll gain a perk point every time you level up, which **can be refunded at any time** through this menu.`
`Perks and techniques **cannot be refunded** or disabled once learned, consider your choices carefully.`

Are they not refundable then? 🤔